app/scripts/controllers/alert-controller.test.ts

```diff
@@ -0,0 +1,210 @@
+/**
+ * @jest-environment node
+ */
+import { ControllerMessenger } from '@metamask/base-controller';
+import { KeyringControllerStateChangeEvent } from '@metamask/keyring-controller';
+import { SnapControllerStateChangeEvent } from '@metamask/snaps-controllers';
+import { EthAccountType } from '@metamask/keyring-api';
+import {
+  AlertControllerActions,
+  AlertControllerEvents,
+  AlertController,
+  AllowedActions,
+  AllowedEvents,
+} from './alert-controller';
+
+const EMPTY_ACCOUNT = {
+  id: '',
+  address: '',
+  options: {},
+  methods: [],
+  type: EthAccountType.Eoa,
+  metadata: {
+    name: '',
+    keyring: {
+      type: '',
+    },
+    importTime: 0,
+  },
+};
+describe('alert controller', () => {
```

Comment:
Nit: to align with other controller tests
```suggestion
describe('AlertController', () => {
```